civet.dev/public/playground.worker.js

```diff
@@ -1,30 +1,48 @@
 importScripts('https://unpkg.com/prettier@2.8.1/standalone.js');
 importScripts('https://unpkg.com/prettier@2.8.1/parser-typescript.js');
+importScripts('https://unpkg.com/shiki');
 importScripts('/__civet.js');
 
-onmessage = (e) => {
-  const { uid, code } = e.data;
+onmessage = async (e) => {
+  const { uid, code, prettierOutput } = e.data;
+  const highlighter = await getHighlighter();
+  const inputHtml = highlighter.codeToHtml(code, { lang: 'coffee' });
```

Comment:
You can also use https://github.com/DanielXMoore/Civet/blob/master/lsp/syntaxes/civet.json - https://github.com/shikijs/shiki/blob/main/docs/languages.md#supporting-your-own-languages-with-shiki to use your vscode tooling here

Reply:
Awesome idea! Going to implement :rocket: Thank you!